gossip/src/crds_value.rs

```diff
@@ -494,80 +499,125 @@ impl Sanitize for NodeInstance {
 pub struct RestartLastVotedForkSlots {
     pub from: Pubkey,
     pub wallclock: u64,
-    pub slots: Vec<CompressedSlots>,
+    pub compressed_slots: Vec<u8>,
```

Comment:
This is a blob of compressed bytes. Why is it public?

Reply:
Changed.